### ISSUE TITLE
Project doesn't build

### ISSUE THREAD

**Issue by calle482** [davidonete/mangosbot-docker#2]
Hi, I've tried building this project today (2024-03-17) from the main branch and it doesn't seem to build anymore. From what I can make out it seems something have changed in the bot source code. I've included an error log below.

I'm running the default values from the docker-compose file, set up the permissions according to the guide and downloaded the vanilla data from the mega link.

https://pastebin.com/F9V69w7B

**Comment by flekz-games** [comment]
Yes many things changed recently on the code, I'll fix it once I get a chance

**Comment by Daxiongmao87** [comment]
wondering if it'd be worth exploring the idea of having different branches/tags here that would have the dockerfiles clone to a specific commit that way those branches should always work, while keeping main/master bleeding edge albeit volatile.

Edit: I guess in the mean time we could set the ENV_CORE_COMMIT_HAS/$CORE_COMMIT_HASH, but that looks like it's only for the mangos-classic repo
